public/js/main.js

```diff
@@ -71,9 +71,12 @@ const initCamera = async () => {
   }
 
   try {
+    const frameRate =
+      state.backend === 'cpu' ? { ideal: 10, max: 20 } : { ideal: 20, max: 30 };
```

Comment:
I remember when I ran a small test with cpu backend, handpose consistently got around 2 fps - so consider using something lower for cpu ideal.